src/MeetingsSDKAdapter.test.js

```diff
@@ -248,37 +258,46 @@ describe('Meetings SDK Adapter', () => {
     beforeEach(() => {
       meetingSDKAdapter.meetings[meetingID] = {
         ...meeting,
-        localVideo: {
-          getVideoTracks: jest.fn(() => [{enabled: true}]),
-        },
+        localVideo: {},
       };
     });
 
-    test('mutes video if the the video track is enabled', () => {
-      meetingSDKAdapter.handleLocalVideo(meetingID);
+    test('mutes video if the the video track is enabled', async () => {
+      await meetingSDKAdapter.handleLocalVideo(meetingID);
       expect(mockSDKMeeting.muteVideo).toHaveBeenCalled();
     });
 
-    test('emits the custom event after muting the video track', () => {
-      meetingSDKAdapter.handleLocalVideo(meetingID);
+    test('disabledLocalVideo property should be defined once the video track is muted', async () => {
```

Comment:
Why are you checking for `disabledLocalVideo`, we should check for `localVideo` since that's what the interface defines